### PR TITLE
Sorting of questionnaires

### LINKED ISSUE #172: Easier sorting of questionnaires
Currently, questionnaires are sorted by an index, which is an attribute of each questionnaire. This attribute should not need to be edited directly, but through a drag'n'drop GUI.

### PR DESCRIPTION
**Pull request by Bartzi**
fixes #172 
and addresses all comments made in #490.

There is just one thing I did not do:
* I did not delete the hidden field rendering in ```faq_index.html``` and ```faq_section.html``` because they do not use the ```as_div``` function declared in the BootstrapMixin.

### CODE REVIEW

File: evap/evaluation/models.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from django_fsm.db.fields import FSMField, transition
 import django.dispatch
 from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin, Group
+from django import forms
```
Comment (karyon) [inline evap/evaluation/models.py]: please remove that again

File: evap/static/js/csrf_utils.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		for (var i = 0; i < cookies.length; i++) {
+			var cookie = jQuery.trim(cookies[i]);
+			// Does this cookie string begin with the name we want?
+			if (cookie.substring(0, name.length + 1) == (name + '=')) {
+				cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+				break;
+			}
+		}
+	}
+	return cookieValue;
```
Comment (karyon) [inline evap/static/js/csrf_utils.js]: you could just `return decodeURIFoo()` inside the loop.

also, is it really that complicated to retrieve some cookie value? :) (i honestly don't know)

File: evap/staff/templates/staff_course_review.html [hunk 3]
```diff
@@ -27,9 +27,6 @@ <h3 class="panel-title">
                     </h3>
                 </div>
                 <div class="panel-body">
-                    {% for hidden in form.hidden_fields %}
-                        {{ hidden }}
-                    {% endfor %}
```
Comment (karyon) [inline evap/staff/templates/staff_course_review.html]: what about the other forms where we render hidden like this, should they be changed as well?

Comment (Bartzi) [inline evap/staff/templates/staff_course_review.html]: They should not be changed as they do not use the ```as_div``` method of the ```BootstrapMixin```

**Comment by Bartzi** [comment]
okay I made changes that address all comments :tada:

**Comment by karyon** [comment]
please rebase and then :ship:

**Pull request merged**